habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -261,39 +276,69 @@ def reconfigure(self, config: Config):
                 ao: ao.joint_positions for ao in self.art_objs
             }
 
+    def set_robot_base_to_random_point(self):
+        for _ in range(50):
```

Comment:
What happens if we try 50 times and still don't succeed? Should there be an assertion or default behavior? Should we sample endlessly and log every 50 attempts?

Reply:
No, this could kill a long training job. I think it is better to just take a starting position with a collision instead (even if it is silent). I think logging if this happens is reasonable though, added.